docs/en/sql-reference/statements/select/order-by.md

```diff
@@ -221,3 +221,70 @@ returns
 │ 1970-03-12 │ 1970-01-08 │ original │
 └────────────┴────────────┴──────────┘
 ```
+
+## OFFSET FETCH Clause {#offset-fetch}
+
+`OFFSET` and `FETCH` allow you to retrieve just a portion of the rows that are generated by the rest of the query.
+
+``` sql
+OFFSET offset_row_count {ROW | ROWS} FETCH {FIRST | NEXT} fetch_row_count {ROW | ROWS} {ONLY | WITH TIES}
+```
+
+The `FETCH` is an alternative to the [LIMIT] (../../../sql-reference/statements/select/limit.md) clause and retrieves rows from a query `SELECT` type.
+
+The `OFFSET` says to skip that many rows before beginning to return rows.
+
+For example, the following query
+
+``` sql
+SELECT * FROM test_fetch ORDER BY a OFFSET 1 ROW FETCH FIRST 3 ROWS ONLY
+```
+
+is identical to the query
+
+``` sql
+SELECT * FROM test_fetch ORDER BY a LIMIT 3 OFFSET 1
+```
+
+When using `FETCH`, it is important to use an [ORDER BY] (../../../sql-reference/statements/select/order-by.md) clause that constrains the result rows into a unique order. Otherwise, you will get an unpredictable subset of the query's rows.
+
+`ROW` and `ROWS` as well as `FIRST` and `NEXT` are noise words that don't influence the effects of these conditions.
+
+The `ONLY` option is used to return rows that immediately follow the rows omitted by the `OFFSET`.
+
+The `WITH TIES` option is used to return any additional rows that tie for the last place in the result set according to the `ORDER BY` clause. The `ORDER BY` is mandatory in this case.
```

Comment:
ORDER BY is mandatory when you use it indeed. Does this sentece mean that OFFSET FETCH can be used as a separate clause and it doesn't neet ORDER BY at all? If OFFSET FETCH is independent from ORDER BY, why are you desribing it in the ORDER BY section?